### PR TITLE
draw bullets as rectangles

### PR DESCRIPTION
**Pull request by a-sep**

### CODE REVIEW

File: js/bullets.js [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
-    // Go through all game objects and check if bullet's x and y coordinates
-    // are same any of the object, you can assume hitbox to be a rectangle
-    // even if the enemies are looking funky
-    // TODO
-  }
+    this.detectCollision = function() {
+        // Go through all game objects and check if bullet's x and y coordinates
+        // are same any of the object, you can assume hitbox to be a rectangle
+        // even if the enemies are looking funky
+        // TODO
```
Comment (Trion129) [inline js/bullets.js:29]: Enemies and Spaceships are completed now, you can implement the detectCollision function now

File: js/bullets.js [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
-var Bullet = function(x, y, up){
-  this.pos = createVector(x, y);
+var Bullet = function(x, y, up) {
+    this.pos = createVector(x, y);
 
-  //Creates acceleration upwards or downwards, if up
-  this.acc = createVector(0, BulletForce * (up ? -1 : +1 ) );
+    //Creates acceleration upwards or downwards, if up
+    this.acc = createVector(0, BulletForce * (up
+        ? -1
```
Comment (Trion129) [inline js/bullets.js:8]: This is bad coding style and can cause difficulty in reading as the operation is spread to 3 lines

**Review by Trion129 — CHANGES_REQUESTED** [review]

File: js/bullets.js [hunk 3]
```diff
@@ -1,25 +1,31 @@
 var BulletForce = 5;
```
Comment (Trion129) [inline js/bullets.js:1]: I guess it should be moved to global variables file

**Review by Trion129 — CHANGES_REQUESTED** [review]

File: js/bullets.js (hunk 3 above)
Comment (a-sep) [reply js/bullets.js:1]: ok

File: js/bullets.js (hunk 2 above)
Comment (a-sep) [reply js/bullets.js:8]: this is result of using atom-beautify plugin, I have to turn it off :)

**Comment by Trion129** [comment]
Merging now, make a new pull request afterwards for detectCollision

**Pull request merged**